e2e/e2e.rs

```diff
@@ -0,0 +1,2 @@
+#[cfg(test)]
+mod tests;
```

Comment:
The tests directory doesn't have a `mod.rs`. My unverified assumption has been that that's needed, so just want to hint at this to ensure the tests are actually run.

Reply:
https://github.com/openshift/cincinnati/pull/249/commits/7dadb9960ae88c23249c521f7e2e223ed5fdf6c0 reworks this to make e2e tests binary have `e2e-...` name (previously it had `mod-...`).

[e2e test results](https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_cincinnati/249/pull-ci-openshift-cincinnati-master-e2e/385/artifacts/e2e/container-logs/test.log):
```
 tests
test e2e_channel_success::channel_b_amd64 ... ok
test e2e_channel_success::channel_a_amd64 ... ok
test e2e_channel_success::channel_test_amd64 ... ok
```